overlays/haskell.nix

```diff
@@ -551,10 +552,50 @@ final: prev: {
                 ) rawProject.hsPkgs
                 // {
                   # These are functions not packages
-                  inherit (rawProject.hsPkgs) shellFor makeConfigFiles ghcWithHoogle ghcWithPackages;
+                  inherit (rawProject.hsPkgs) makeConfigFiles ghcWithHoogle ghcWithPackages;
+                  shellFor = a: __trace "project.hsPkgs.shellFor has been deprecated in favour of project.shellFor"
```

Comment:
:+1:

Maybe we should deprecate the other functions in here and move them up a level?

Reply:
Done.